packages/fauna/fauna/resources/indexes/user_by_provider_and_provider_account_id.fql

```diff
@@ -0,0 +1,9 @@
+CreateIndex({
+  name: "user_by_provider_and_provider_account_id",
+  source: Collection("accounts"),
```

Comment:
Since the `source` is `accounts` collection, may be we should name it `account_by_provider_and_provider_account_id`?

Reply:
True. It IS used to retrieve the actual User in the adapter method, but here it should be renamed. Will change!

Reply:
Changed!